tests/playwright/src/specs/kubernetes.spec.ts

```diff
@@ -148,8 +148,57 @@ test.describe('Kubernetes resources End-to-End test', { tag: '@k8s_e2e' }, () =>
     await playExpect(dashboardPage.namespaceDropdownButton).toBeVisible({ timeout: 10_000 });
     await playExpect(dashboardPage.currentNamespace).toHaveValue('default');
 
+    await playExpect
```

Comment:
Actual test case name does not fit anymore, rename to something like K8s Dashboard metrics and namespace test.